var/spack/repos/builtin/packages/intel-mpi/package.py

```diff
@@ -22,15 +22,23 @@
 # License along with this program; if not, write to the Free Software
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 ##############################################################################
-import os
 
 from spack import *
-from spack.environment import EnvironmentModifications
 
 
 class IntelMpi(IntelPackage):
     """Intel MPI"""
 
+    # BTW:
+    # https://software.intel.com/en-us/articles/intel-mpi-library-release-notes-linux
+    #   Intel MPI Library 2018 Update 2
+    #   ...
+    #   Intel MPI Library is now available .. in YUM and APT repositories.
+    #   ...
+    # See also:
+    # https://software.intel.com/en-us/articles/installing-intel-free-libs-and-python-yum-repo
+    # https://software.intel.com/en-us/articles/installing-intel-parallel-studio-xe-on-aws-linux-instances
```

Comment:
I don't know if this note is necessary. Most packages in Spack are available through yum and apt, no need to advertise this fact.